prov/efa/src/efa_av.c

```diff
@@ -1141,17 +1086,18 @@ void delete_efa_resource(struct efa_resource *resource) {
 void test_duplicate_efa_ah_creation() {
 	/* Setup Local Variables */
 	int ibv_err = 4242;
-	struct efa_resource *resource;
+	struct efa_resource resource = {0};
+	int ret;
 	struct efa_av *av;
 	struct efa_ah *efa_ah;
 	struct efa_ah *efa_ah2;
 	uint8_t gid[EFA_GID_LEN];
 	memset(gid, 7, EFA_GID_LEN);
 
 	will_return(__wrap_efadv_query_device, 0);
-	resource = create_efa_resource();
-	assert_non_null(resource);
-	av = container_of(resource->av, struct efa_av, util_av.av_fid);
+	ret = efa_resource_create(&resource);
```

Comment:
According to our naming convention https://github.com/ofiwg/libfabric/blob/main/prov/efa/docs/naming_convention.md, this functions should be named `efa_resource_construct`.

Reply:
also may be we source add `test_` prefix to the function.

Reply:
Done